src/icp_v2.cc

```diff
@@ -23,6 +23,7 @@
 #include "comm/Connection.h"
 #include "comm/Loops.h"
 #include "fd.h"
+#include "fde.h"
```

Comment:
> `error: fd_table was not declared in this scope`

How does fd_table get into src/icp_v2.cc code? I do not see any direct use or any FD_READ_METHOD()/FD_WRITE_METHOD() calls in src/icp_v2.cc, but my build environment is different. I am wondering whether we should `#include` this header somewhere else instead (to prevent not just this, but similar build errors in the future).